frontend/add-client/form-inputs/demographic-information-inputs.component.tsx

```diff
@@ -113,7 +113,6 @@ export default function DemographicInformationInputs(
             type="number"
             value={householdSize}
             onChange={evt => setHouseholdSize(Number(evt.target.value))}
-            required
```

Comment:
Why are we making household size optional? We should only be making date of US arrival optional.